addons/mail/static/src/owl/components/composer/composer.js

```diff
@@ -306,14 +307,19 @@ class Composer extends Component {
      * @param {string} ev.detail.unicode
      */
     _onEmojiSelection(ev) {
-        this._textInputRef.comp.insertTextContent(ev.detail.unicode);
+        this._saveComposerContent();
+        this.storeDispatch(
+            'insertEmoji',
+            ev,
```

Comment:
I would directly pass the emoji (unicode) instead of the event.

Reply:
Done